### ISSUE TITLE
Unable to collapse spawn eggs into a REI category

### ISSUE THREAD

**Issue by SplendidAlakey** [Queerbric/Inspecio#108]
**MC** 1.19.2
**Inspecio** 1.6.0
**REI** 9.1.619

I'm unable to group spawn eggs together in REI, if Inspecio is installed. Could be affecting other items, but I can only confirm the issue to be present with the eggs.

I use KubeJS to reproduce this, but grouping can also be done via mods:

1. Create the following KubeJS script in `kubejs/client_scripts`:
```
// priority: 9

REIEvents.groupEntries((event) => {
	
// Group spawn eggs together
	
	event.groupItems("kubejs:rei_groups/spawn_eggs", "Spawn Eggs", [
		'/spawn_egg/'
	]);
	
});
```
2. Run the game and look at spawn eggs in REI - they aren't grouped together
3. Remove Inspecio and launch the game again - now the eggs are grouped up

This happens even if spawn egg tooltips are disabled in the config.

Let me know if this needs to be reported to either REI or KubeJS, instead.

Logs: https://gist.github.com/SplendidAlakey/16ef59aa89fb590c6d9ddbbff09fee30

**Comment by LambdAurora** [comment]
What, did you check if it happened with/without QSL?

I don't see how this mod could have anything to do with it.

**Comment by SplendidAlakey** [comment]
Yep. In fact, the Fabric port [Advanced Tooltips](https://www.curseforge.com/minecraft/mc-mods/advanced-tooltips) has the same issue.

**Comment by SplendidAlakey** [comment]
After looking a bit at the logs, I decided to also report this to KubeJS, as it seems to have something to do specifically with it: https://github.com/KubeJS-Mods/KubeJS/issues/633
